src/main/java/Calculator/Calculator.java

```diff
@@ -0,0 +1,64 @@
+package Calculator;
+
+public class Calculator {
+    String inputString;
+    int result;
+
+    public Calculator(String inputString) {
+        if (inputString.isEmpty())
+            throw new IllegalArgumentException("입력 값이 null이거나 빈 공백 문자입니다.");
+
```

Comment:
```suggestion
        if (inputString.isEmpty()) {
            throw new IllegalArgumentException("입력 값이 null이거나 빈 공백 문자입니다.");
        }

```
한 줄이라도 블록을 생략하지 않는 것이 좋습니다.
코드가 추가되었을 때 블록을 놓치는 실수를 미리 방지할 수 있기도 하고 코드 일관성도 갖출 수 있어요!